src/encoded/region_indexer.py

```diff
@@ -388,6 +390,18 @@ def get_from_es(request, comp_id):
         '''Returns composite json blob from elastic-search, or None if not found.'''
         return None
 
+    def update_objects(self, request, uuids, force):
```

Comment:
Update objects was move to region and vis indexer child classes because snovault indexer class has a different update objects now.